superbuild/projects_modules/openssl.cmake

```diff
@@ -92,7 +92,7 @@ else (WIN32)
     UPDATE_COMMAND ""
     PATCH_COMMAND ""
     DEPENDS ${${ep}_dependencies}
-    CONFIGURE_COMMAND ${EP_PATH_SOURCE}/${ep}/config no-zlib shared --prefix=${build_path} --openssldir=${build_path} 
+    CONFIGURE_COMMAND ${EP_PATH_SOURCE}/${ep}/config no-zlib  shared --prefix=${build_path} --openssldir=${build_path} 
```

Comment:
Hmm I don't know the configure step, but shouldn't it be 'static' instead of 'shared' ?
I don't think the `BUILD_SHARED_LIBS OFF` is enough to create the static lib.

Reply:
It is ok static lib are generated (the makefile is not generated by cmake) and without shared I got compilation issues.

Reply:
ok replace by no-shared